src/locales/en/translation.json

```
@@ -34,12 +34,21 @@
       "confirm_all": "Confirm all",
       "zoom_to_attribute": "Zoom to attribute"
     },
-    "growth_form": "Growth form"
+    "growth_form": "Growth form",
+    "feature_introduction": {
+      "title": "MERMAID just got smarter",
+      "description": "Through a partnership with <a href=\"/\" target=\"_blank\">CoralNet<\/a> from the University of California San Diego and Scripps Institution of Oceanography, MERMAID now includes AI-based classification using a pyspacer deep learning model. This integration enables fast, scalable analysis of benthic photos so that you can generate insights quickly and consistently with machine learning and AI.",
```

This doesn't look as clean with `\`
```suggestion
      "description": "Through a partnership with <a>CoralNet</a> from the University of California San Diego and Scripps Institution of Oceanography, MERMAID now includes AI-based classification using a pyspacer deep learning model. This integration enables fast, scalable analysis of benthic photos so that you can generate insights quickly and consistently with machine learning and AI.",
```